docs/data-companion/pruning.md

```diff
@@ -0,0 +1,346 @@
+---
+order: 1
+parent:
+    title: Pruning Service
+    order: 3
+---
+
+# Pruning data via the pruning service
+
+CometBFT employs a sophisticated pruning logic to eliminate unnecessary data and reduce storage requirements.
+
+This document covers use cases where the pruning process on a CometBFT node can be influenced via the Data Companion
+pruning service API.
+
+The pruning process runs asynchronously in the background based on the retain
+heights set either by the application or via the Data Companion API. Before the
+Data Companion API was introduced, that is, before v0.38, pruning could be
```

Comment:
For all effects, this is v0.38, right? Moreover, this is not enabled in the actual (not experimental) v0.38.

Reply:
```suggestion
Data Companion API was introduced, that is, in v0.38 and earlier versions, pruning could be
```